notebooks/page view functional network/page_view_approach_economic_recovery.ipynb

```diff
@@ -0,0 +1,1508 @@
+   "metadata": {},
+   "source": [
+    "### Order by session hits"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sessions = list(sorted(economic_recovery_subgraph, key=lambda x: x[1]['sessionHitsAll'], reverse=True))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Order by shortest page path lengths"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# calculate the paths of the shortest page path\n",
+    "shortest_paths_seed0 = nx.shortest_path_length(g, source='/browse/working/finding-job')\n",
+    "shortest_paths_seed1 = nx.shortest_path_length(g, source='/topic/further-education-skills/apprenticeships')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# seeed0.1: shortest_paths_seed0.1: '/browse/working/finding-job'\n",
+    "shortest_seed0_1 = {k: v for k, v in sorted(shortest_paths_seed0.items(), key=lambda item: item[1], reverse=False)}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# seed0.2: shortest_paths_seed0.2: '/topic/further-education-skills/apprenticeships'\n",
+    "shortest_seed0_2 = {k: v for k, v in sorted(shortest_paths_seed1.items(), key=lambda item: item[1], reverse=False)}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Order by the sum of the shortest page path lengths"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "combined_paths = []\n",
+    "\n",
+    "for page, closeness in finding_job.items():\n",
+    "    for page1, closeness1 in apprentice.items():\n",
+    "        if page == page1: \n",
+    "            combined_score = closeness + closeness1\n",
+    "            combined_paths.append(page)\n",
+    "            combined_paths.append(combined_score)\n",
+    "            \n",
+    "combined_paths=dict(zip(combined_paths[::2], combined_paths[1::2]))\n",
+    "combined_paths={k: v for k, v in sorted(combined_paths.items(), key=lambda item: item[1], reverse=True)}\n",
+    "combined_paths"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Order by centrality metrics"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# between centrality: the number of shortest paths that pass through the node - a 'bridge' between nodes\n",
+    "# higher values indicate higher centrality\n",
+    "g_di = nx.DiGraph(g)\n",
+    "g_di_between = nx.betweenness_centrality(g_di)\n",
+    "g_between = dict(sorted(g_di_between.items(), key=operator.itemgetter(1),reverse=True))\n",
```

Comment:
Since you wrote `from operator import itemgetter` earlier on in the code, you can just write `key=itemgetter(1)` here.
But, you could also write it as this:
`key = lambda x: x[1]`
itemgetter should be a bit faster than the lambda function though.

Reply:
Thanks again, resolved in commit 287f84a